.eslintrc

```diff
@@ -1,16 +1,13 @@
 {
   "parserOptions": {
-    "ecmaVersion": 6,
-    "sourceType": "module"
-  },
-  "ecmaFeatures": {
-    "modules": true
+    "sourceType": "module",
+    "ecmaFeatures": {
+      "impliedStrict": true
+    }
   },
   "env": {
-    "node": true
-  },
-  "globals": {
-
+    "node": true,
+    "es2020": true
```

Comment:
es2020 is still not supported in Node 10. I hesitate to add linting support since it might lead to us introducing features that depend on newer JS that is not yet supported in our runtime.

https://node.green/#ES2020

Reply:
@mattolson
Actually, Node will just break when encountering unsupported syntax, so if we have good test coverage (or at the least test our changes manually before releasing) - we won’t miss these parts.
But I can just set "es2018" for the linter, it seems to have good support by Node 10 and includes all the needed features (esp. object rest/spread operator).

Reply:
Since this is 2.x branch we will also need to support node 8 here since this is used by stapler which is currently running on node 8.

Reply:
Updated the config and set `"ecmaVersion": 2018`. It's fully supported by Node 10 and partially (but it's enough) supported by node 8